features/doctrine/date_filter.feature

```diff
@@ -386,204 +386,25 @@ Feature: Date filter on collections
     Then the response status code should be 200
     And the response should be in JSON
     And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
-    And the JSON should be equal to:
+    And the JSON should be valid according to this schema:
```

Comment:
If we're fixing this, it should be done in another PR, I think.

(And the `?exists[exists]` below is from before...)